vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -4,30 +4,36 @@ sidebarDepth: 2
 # Export and Publish a Bundle
 
 ## Overview
-Use the Entando bundler command to export a bundle of Entando components from an existing Entando application. An Entando bundle can be used to do the initial install of Entando components into an Entando application, migrate Entando components from one environment to another (e.g. Dev to QA), to provide a template for building a new Entando application, or as the skeleton of an Entando solution. The output of this command is the same bundle directory structure created by an Entando project including a bundle descriptor file.
+Use the bundler command to export a bundle of Entando components from an existing Entando Application. An Entando Bundle can be used 
+- For the initial install of Entando components into an Entando Application
+- To migrate Entando components from one environment to another (e.g. Dev to QA)
+- To provide a template for building a new Entando Application
+- As the skeleton of an Entando solution
+
+This command generates the same bundle directory structure as an Entando project, including a bundle descriptor file.
 
 ### Prerequisites
-* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify you have the prerequisites in place for this tutorial (e.g. git, entando-bundler). 
+* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify prerequisites (e.g. git, entando-bundler): 
 ``` sh
 ent check-env develop 
 ```
-* You'll also need a running Entando application.
-* You'll need admin access to Keycloak or admin access to Kubernetes to set it up.
+* A running Entando Application
+* Admin access to Keycloak or admin access to Kubernetes to set it up
 
-The tutorial assumes you're using an Entando quickstart application. You may need to adjust the specific URLs, credentials, namespaces, etc. for a custom application.
+This tutorial assumes you're using an Entando quickstart application. You may need to adjust specific URLs, credentials, namespaces, etc., for a custom application.
 
 ## Export an Entando Bundle
 
-### Setup the Keycloak client
-You'll need to setup a Keycloak client with the appropriate permissions for the bundler to access all of the necessary Entando APIs.
+### Setup the Keycloak Client
+Configure a Keycloak client to grant the bundler to access to the required Entando APIs.
 
-1. Find the Secret for the Keycloak admin account. If you already have the admin credentials, then you can skip to step 3. 
+1. Find the Secret for the Keycloak admin account: 
 ``` sh
 kubectl get secrets -n entando 
 ```
-In a quickstart application, the Secret is named.`default-sso-in-namespace-admin-secret`
+In a quickstart application, the Secret is named `default-sso-in-namespace-admin-secret`. If you already have the admin credentials, skip to Step 3.
```

Comment:
I think the skip step should be ahead of the command, so they don't waste time looking for the secret

Should namespace be YOUR-NAMESPACE?

Reply:
? you still need to find the Secret. i think the flow is logical. the optional bit is whether or not you need to look up credentials to access the Secret you just found

Reply:
and pretty sure the name is as-is

Reply:
I just realized the duplication we have here. Maybe we can make admin access to keycloak a prereq, and just point to https://developer.entando.com/v7.0/docs/consume/identity-management.html#logging-into-your-keycloak-instance as I noted above? That cuts out a number of steps here.